packages/common/src/core/types/IHubGroup.ts

```diff
@@ -119,3 +122,15 @@ export interface IHubGroup extends IHubEntityBase, IWithPermissions {
    */
   _clearEmptyFields?: boolean;
 }
+
+/**
+ * This type redefines the IHubGroup interface in such a way
+ * that it can be consumed by the entity editor.
+ */
+export type IHubGroupEditor = Omit<IHubGroup, "extent"> & {
```

Comment:
Since `IHubGroup` does not have an `extent`, you could just do `export type IHubGroupEditor = IHubGroup & { ...`